core/server/api/v2/index.js

```diff
@@ -1 +1,7 @@
-module.exports = {};
+const shared = require('../shared');
+
+module.exports = {
+    get http() {
+        return shared.http;
```

Comment:
it's used in two fn's here, so i decided to have a global export. i don't think it doesn't rly matter, because the shared package uses magic getters anyway